apps/condo/bin/generate-initial-data.js

```diff
@@ -0,0 +1,30 @@
+const path = require('path')
+const { GraphQLApp } = require('@keystonejs/app-graphql')
+
+class InitialDataGenerator {
+    async connect () {
+        const resolved = path.resolve('./index.js')
+        const { distDir, keystone, apps } = require(resolved)
```

Comment:
Could you please use prepareKeystoneApp from test utils

Reply:
https://github.com/open-condo-software/condo/blob/master/apps/condo/bin/test-sbbol-oauth-by-hands.js#L22 Example

Reply:
Done